### PR TITLE
Added API Route for POST user details

### PR DESCRIPTION
**Pull request by pranavgade20**
added API Route for POST user details for Lead Management Server
GCI Task Link: https://codein.withgoogle.com/dashboard/task-instances/4975580924608512/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,10 +17,12 @@
...
-    "express": "^4.15.3",
-    "firebaseui": "^2.5.1"
-    "socket.io": "^2.0.4"
+    "angular-patternfly": "^4.18.6",
+    "express": "^4.16.4",
+    "firebase": "^5.5.5",
+    "firebaseui": "^3.4.1",
+    "mongodb": "^3.1.8",
+    "socket.io": "^2.1.1"
   },
```
Comment (nikitadhiman) [inline package.json]: These changes should be in separate PR

Comment (pranavgade20) [reply package.json]: > These changes should be in separate PR

But they are related to this PR

**Comment by pranavgade20** [comment]
@nikitadhiman Where should I use Promises?
Because I don't think promises are required, as the code has maximum two nested callbacks.

File: package.json (hunk 1 above)
Comment (nikitadhiman) [reply package.json]: These are separate tasks. Even thou you might have updated dependencies we don't need to include them in this PR. More updates coming soon from our end and hence this might create conflict thus it is adviced to separate them out. When we check...we check in latest updated environment dependencies

**Comment by nikitadhiman** [comment]
Actually
